components/Servers.js

```diff
@@ -65,7 +65,10 @@ function ServerVariables({ variables }) {
   );
 }
 
-function ServerSecurity({ security, asyncapi }) {
+function ServerSecurity({ protocol, security, asyncapi }) {
+  if (isKafka(protocol)) {
+    return KafkaServerSecurity({ protocol, security, asyncapi });
+  }
```

Comment:
`isKafka` function is used only in one place, should it can be inlined. Also I know that `KafkaServerSecurity` can be used as normal JS function, but we should treat it as JSX component:

 ```suggestion
  if (protocol === 'kafka' || protocol === 'kafka-secure') {
    return <KafkaServerSecurity protocol={protocol} security={security} asyncapi={asyncapi} />
  }
```

and then please remove `isKafka` function.